js/admin-api-endpoints.js

```
@@ -0,0 +1,738 @@
+  if (!currentModal) return;
+
+  currentModal.activeTab = tabName;
+  const response = currentModal.response;
+
+  switch (tabName) {
+    case 'formatted':
+      const formattedData = typeof response.data === 'object'
+        ? JSONFormatter.format(response.data)
+        : response.data;
+
+      content.innerHTML = `
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)